app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("./cinema.sqlite")
+        self.table_name = "actors"
+        self.db_name = "cinema.sqlite"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"""
+            INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)

```

Comment:
Check sql formatting - each key word must start from a new line. Use this tool to help you with styling https://sqlformat.org/